addons/website_blog/migrations/9.0.1.0/post-migration.py

```diff
@@ -27,11 +27,10 @@ def convert_blog_post_cover(env):
         post = post_obj.browse(row[0])
         post.cover_properties = (
             '{{"background-image": "url({})",'
-            ' "opacity": "1",'
+            ' "opacity": "0",'
```

Comment:
in 10, it's the other way around. 10 creates a div that overlays the image, and opacity 1 combined with color none (=white) will give you just blank: https://github.com/OCA/OpenUpgrade/blob/10.0/addons/website_blog/views/website_blog_templates.xml#L185
Hmmm, change it back for 9 just to have to change it back again for 10. Would be cleaner...